### PR TITLE
Add Ukraine map

### PR DESCRIPTION
**Pull request by i-iurchuk**
@VictorCazanave Hi) Review please)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ A set of SVG maps that can be installed via [npm](https://www.npmjs.com) or [yar
 * 🇹🇼[Taiwan, Main Island](packages/taiwan.main)
 * 🇹🇳[Tunisia](packages/tunisia)
 * 🇦🇪[UAE](packages/uae)
+* 🇦🇪[Ukraine](packages/ukraine)
```
Comment (VictorCazanave) [inline README.md]: Could you use the right flag 🇺🇦please?

File: packages/ukraine/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# 🇺🇦 Ukraine
+
+Contains all the states and the Federal District of Ukraine:
```
Comment (VictorCazanave) [inline packages/ukraine/README.md]: I think these are the regions, aren't these? 🤔

File: packages/ukraine/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Poltava
+* Rivne
+* Sevastpol City
+* Sumy
+* Ternopil
+* Transcarpathia
+* Vinnytsya
+* Volyn
+* Zaporizhzhya
+* Zhytomyr
```
Comment (VictorCazanave) [inline packages/ukraine/README.md]: Could you use the English names (in the SVG file too) please?

File: packages/ukraine/package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/VictorCazanave/svg-maps/tree/master/packages/urkaine"
+  },
+  "keywords": [
+    "svg",
+    "map",
+    "ukraine"
+  ],
+  "author": "Victor Cazanave <victor.cazanave@gmail.com>",
```
Comment (VictorCazanave) [inline packages/ukraine/package.json]: You can use your name 😄

**Comment by i-iurchuk** [comment]
I saw another pull request for Ukraine, so I’ll close mine)

**Pull request closed without merging**